tests/pipelines/test_pipelines_common.py

```diff
@@ -59,6 +61,107 @@ def check_same_shape(tensor_list):
     return all(shape == shapes[0] for shape in shapes[1:])
 
 
+class IPAdapterTesterMixin:
```

Comment:
Please let me know your thoughts on if this has been done as you expected. Pipelines that support IPAdapter's should derive from this and create dummy image_encoder and feature_extractor instead of `None`.